scripts/tests.js

```diff
@@ -4,63 +4,49 @@ process.env.NODE_ENV = 'test';
 process.env.PUBLIC_URL = '';
 
 const jest = require('jest');
-const fs = require('fs');
-const fsx = require('fs-extra');
+const fs = require('fs-extra');
 const path = require('path');
 
-function dirExists(dirPath) {
-  const F_OK = (fs.constants && fs.constants.F_OK) || fs['F_OK'];
-  try {
-    fs.accessSync(dirPath, F_OK);
-    return fs.statSync(dirPath).isDirectory();
-  } catch (e) {
-    return false;
-  }
-}
-
 function getDirectories(rootDir) {
   return fs.readdirSync(rootDir).filter(function(file) {
     return fs.statSync(path.join(rootDir, file)).isDirectory();
   });
 }
 
-function getFiles(rootDir) {
-  return fs.readdirSync(rootDir).filter(function(file) {
-    return !fs.statSync(path.join(rootDir, file)).isDirectory();
-  });
-}
-
-function getIntegrationMockContent(file) {
-  return `module.exports = require('../../../../${file}');`;
-}
-
 function createIntegrationMock() {
   const testsRoot = 'tests';
   const testCaseFolders = getDirectories(testsRoot).filter(function(testDir) {
     return !(testDir.startsWith('__') && testDir.endsWith('__'));
   });
-  for (let i = 0; i < testCaseFolders.length; i++) {
-    const testCaseNodeModules = path.join(
-      testsRoot,
-      testCaseFolders[i],
-      'node_modules'
-    );
-    if (!dirExists(testCaseNodeModules)) {
-      fs.mkdirSync(testCaseNodeModules);
-    }
+
+  testCaseFolders.forEach(directory => {
+    const testCaseNodeModules = path.join(testsRoot, directory, 'node_modules');
+
+    const rootDir = path.resolve('.');
     const testCaseModuleFolder = path.join(testCaseNodeModules, 'ts-jest');
-    fsx.copySync('.', testCaseModuleFolder, {
```

Comment:
Had to rewrite this part to comply to new fs-extra api. We can't copy a dir inside itself and then just fix it via filtering, so I'm just being explicit about what we're copying